components/board/fake/board.go

```diff
@@ -283,7 +286,11 @@ func (a *Analog) reset(pin string) {
 	a.Mu.Unlock()
 }
 
+//nolint:gosec
```

Comment:
:grimacing: I don't immediately see the security issue, but I'm pretty strongly against introducing major nolint directives if we can avoid them. Do you understand why the linter was unhappy? I wonder if there's a different way to change it...

Reply:
I'll try and find a different solution